examples/ssd_mobilenet/MobileNet.js

```diff
@@ -0,0 +1,199 @@
+let nnPolyfill, nnNative;
+if (navigator.ml.isPolyfill) {
+  nnNative = null;
+  nnPolyfill = navigator.ml.getNeuralNetworkContext();
+} else {
+  nnNative = navigator.ml.getNeuralNetworkContext();
+  nnPolyfill = navigator.ml_polyfill.getNeuralNetworkContext();
+}
+
+class MobileNet {
+  constructor(tfModel, backend) {
+    this._tfModel = tfModel;
+    this._model = null;
+    this._compilation;
+    this._execution;
+    this._tensorIds = [];
+    this._operandIndex = 0;
+    if (typeof backend !== 'undefined') {
+      this._backend = backend;
+    } else {
+      if (nnNative) {
+        this._backend = 'WebML';
+      } else {
+        this._backend = 'WASM';
+      }
+    }
+    if (this._backend === 'WebML') {
+      if (nnNative === null) {
+        throw Error('Fails to initialize neural network context');
+      }
+      this._nn = nnNative;
+    } else if (this._backend === 'WASM' || this._backend === 'WebGL2') {
+      this._nn = nnPolyfill;
+    }
+  }
+
+  async createCompiledModel() {
+    let options = {};
+    if (this._backend === 'WebGL2') {
+      options.useWebGL2 = true;
+    }
+    this._model = await this._nn.createModel(options);
+
+    this._addTensorOperands();
+    this._addOpsAndParams();
+
+    await this._model.finish();
+    this._compilation = await this._model.createCompilation();
+    this._compilation.setPreference(this._nn.PREFER_FAST_SINGLE_ANSWER);
+    await this._compilation.finish();
+    this._execution = await this._compilation.createExecution();
+  }
+
+  async compute(inputTensor, outputBoxTensor, outputClassScoresTensor) {
+    // this._execution._inputs = [];
+    // this._execution._outputs = [];
+    this._execution.setInput(0, inputTensor);
+    let outH = [1083, 600, 150, 54, 24, 6];
+    let boxLen = 4;
+    let classLen = 91;
```

Comment:
And please document where these constants come from, as you modified the net outputs.